puppet/modules/jenkins_job_builder/files/theforeman.org/unmanaged_jobs.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+require 'net/http'
+require 'json'
+
+url = File.open(ARGV.pop) { |file| file.readlines }.select { |line| line.include?('url=') }.first.gsub(/url=(.*\/\/)?/, '').chomp
```

Comment:
This is a lot to mentally unpack in one line for me, splitting it across a few lines would help readability I think.